### PR TITLE
Fix office hours logic 

### PR DESCRIPTION
**Pull request by lzan98**
Apparently, the issue currently extends to schedule add command as well. 😅Good spot @LeeEnEn!

Screenshot 2022-03-30 at 22 42 38

### CODE REVIEW

File: src/main/java/seedu/address/model/interview/Interview.java [hunk 1]
```diff
@@ -76,14 +76,15 @@ public boolean isDuringOfficeHour() {
         if (this.interviewDay > 5) {
             return false;
         }
-
-        if (interviewHour < 8) {
-            return false;
-        } else if (interviewHour >= 17 && interviewMin > 30) {
+        if (interviewHour < 8 || interviewHour > 17) {
```
Comment (charmainehly) [inline src/main/java/seedu/address/model/interview/Interview.java:79]: @lzan98 hmm just wondering, office hours we stipulated are 8am-6pm, i.e. the last interview can be scheduled at 17:30, will this cause a false return prematurely?

Comment (charmainehly) [inline src/main/java/seedu/address/model/interview/Interview.java:79]: thanks for making the fix!!!

**Review by charmainehly — APPROVED** [review]
LGTM! thanks for catching this!!! :')

**Comment by charmainehly** [comment]
@lzan98 hmm will it be complicated to rework the test cases that i think gradle didn't manage to pass :(

**Comment by lzan98** [comment]
> @lzan98 hmm will it be complicated to rework the test cases that i think gradle didn't manage to pass :(

No issue! There's only one that didn't pass and it was due to a different issue haha

**Comment by lzan98** [comment]
Note: Will add in more test cases for v1.4

**Pull request merged**